src/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="mobile.css" />
```

Comment:
- [ ] 🙏Please fix your style, it is not 🙆‍♀️ working, and it appears that you do not have a file named `mobile.css`. This will enhance the appearance 😁 of your code since only HTML is currently functioning without any CSS.